white_space.py

```diff
@@ -22,24 +21,23 @@ def format_file(path):
     '''
     p = ""
     for dirpath, _, filenames in os.walk(path):
-        if(os.path.exists(os.path.join(dirpath, 'bin/activate'))):
+        if os.path.exists(os.path.join(dirpath, 'bin/activate')):
             p = dirpath
-        for f in filenames:
-            if f.endswith('.py'):
-                filename = os.path.abspath(os.path.join(dirpath, f))
-                if (not bool(p)):
-                    if os.access(filename, os.W_OK):
-                        try:
-                            file1 = open(filename, 'r')
-                            source_code = file1.read()
-                            file1.seek(0)
-                            file1 = open(filename, 'w')
-                            file1.truncate()
-                            file1.write(autopep8.fix_code(source_code))
-                            file1.close()
-                        except:
-                            print(filename)
-                            print(sys.exc_info()[0])
+        for file_ in filenames:
+            if file_.endswith('.py'):
+                file_ = os.path.abspath(os.path.join(dirpath, file_))
```

Comment:
You are reassigning loop variable here, which is bad programming practice.
I suggest using `f` or some other keyword for looping variable, and keep `file_` here as is.